bindings/wasm/examples/src/util.ts

```diff
@@ -15,6 +18,53 @@ export const FAUCET_ENDPOINT = "http://localhost:8091/api/enqueue";
 
 /** Creates a DID Document and publishes it in a new Alias Output.
 
+Its functionality is equivalent to the "create DID" example
+and exists for convenient calling from the other examples. */
+export async function createDidStorage(client: Client, secretManager: SecretManager, storage: Storage): Promise<{
```

Comment:
Should we maybe rename this function? Without a context, it means creating a storage rather than DID. Something like `createIdentity` could work?

Reply:
Yeah, eventually we can just rename it to `createDid` like the other convenience function, but while we transition the examples that name is already taken. So I will do this when we have reworked all examples.